homeassistant/components/met/__init__.py

```diff
@@ -40,10 +40,15 @@ async def async_setup(hass: HomeAssistant, config: Config) -> bool:
 
 async def async_setup_entry(hass, config_entry):
     """Set up Met as config entry."""
-    if (
-        config_entry.data.get(CONF_TRACK_HOME, False)
-        and hass.config.latitude == DEFAULT_HOME_LATITUDE
-        and hass.config.longitude == DEFAULT_HOME_LONGITUDE
+    # Don't setup if tracking home location and latitude or longitude isn't set.
+    # Also, filters out our onboarding default location.
+    if config_entry.data.get(CONF_TRACK_HOME, False) and (
+        not hass.config.latitude
+        or not hass.config.longitude
```

Comment:
We are now blocking anyone living on the equator. Instead we should check if both are 0.

Reply:
Well, move a couple of meters!

You're right though, adjusting 😉